tests/mocks/pyslurm.py

```diff
@@ -867,7 +874,7 @@ def get(self):
 
     def find_id(self, jobid):
         for job in context.CTLD.jobs:
-            if job.jobid == jobid:
+            if str(job.jobid) == str(jobid):
```

Comment:
This is probably useless since all jobids should already be strings at this point.